gateway-js/src/__tests__/executeQueryPlan.test.ts

```diff
@@ -142,6 +142,73 @@ describe('executeQueryPlan', () => {
       expect(response).toHaveProperty('errors.0.extensions.variables', {});
     });
 
+    it(`should not send request to downstream services when all entities are undefined`, async () => {
+      overrideResolversInService('reviews', {
+        User: {
+          reviews() {//We are using an example where a past product was removed from the system
+            return [{id:1,body: 'Love it!', product: undefined}, {id:1,body: 'Love it!', product: { __typename: 'Furniture', upc: '1'}}];
```

Comment:
Not a hard ask, and not sure if this is even a reasonable suggestion, but to avoid bringing/duplciating the fixture data directly into this file (snapshots seems fine, but the data itself might be nice to define in one place), it _might be reasonable_ to `export` the `reviews` from `fixtures/reviews.ts` (perhaps renaming it to `reviewsData` in the process?) and use a transformation on it to set one of the elements to `undefined`.

e.g.,

```typescript
import { reviews } from 'apollo-federation-integration-testsuite';


/* ... down to here ... */

const reviewsData = reviews.reviewsData;

return [
  { ...reviewsData[0], product: undefined },
  { ...reviewsData[1], },
];
```

Reply:
Good call out, the latest push has changes to expose that data set and use it in the test